helpers/messages/sendMessage.ts

```diff
@@ -125,6 +126,8 @@ export async function sendMessage(bot: Bot, channelId: bigint, options: CreateMe
 export interface CreateMessage {
   /** The message contents (up to 2000 characters) */
   content?: string;
+  /** Can be used to verify a message was sent (up to 25 characters). Value will appear in the Message Create event. */
```

Comment:
Does this exist in the DiscordMessage interface and the message transformer?

Reply:
DiscordMessage interface already has nonce.
Added nonce to (Discordeno)Message (return type of message transformer)